README.md

```diff
@@ -1,20 +1,100 @@
 # ConduitMQTT
 
-WIP
+An MQTT adapter for [Conduit](https://github.com/conduitframework/conduit).
+
+## Installation
+
+This package can be installed as:
+
+  1. Add `conduit_mqtt` to your list of dependencies in `mix.exs`:
+
+    ```elixir
+    def deps do
+      [{:conduit_mqtt, "~> 0.1.0"}]
+    end
+    ```
+
+  2. Ensure `conduit_mqtt` is started before your application:
+
+    ```elixir
+    def application do
+      [applications: [:conduit_mqtt]]
+    end
+    ```
+
+## Configuring the Adapter
+
+```elixir
+# config/config.exs
+
+config :my_app, MyApp.Broker,
+  adapter: ConduitMQTT, connection_opts: [server: {Tortoise.Transport.Tcp, host: 'localhost', port: 1883}]  #TODO IS THIS CORRECT WAY TO GET INTO ADAPTER OPTS?
```

Comment:
Yes. Add some newlines.

Reply:
where are newlines valid?  I need the connection_opts to get passed in the opts of the ConduitMQTT adapter not anything higher up that list there